### ISSUE TITLE
V8: Rich Text Editor size

### ISSUE THREAD

**Issue by IainMartin-Ayr** [umbraco/Umbraco-CMS#5565]
The Rich Text Editor doesn't appear to be implementing the dimensions set in the config values for the DataType itself. These are being passed through in the preValues 'page' when these are retrieved for the RTE but not enacted.

`value: {,…}
**dimensions: {height: 200, width: 400}
height: 200
width: 400**
maxImageSize: 500
mode: "classic"`

Here I have deliberately set this to be very small, but the RTE is still the full width, and as you type content in, it's the whole edit pane that is getting the scroll bar and not the RTE - it's height just keeps on getting bigger. 

I checked this on a site I was building when V8 first came out and has no plugins or anything in it and the RTE is growing as you put the content in. Full width is not so much an issue, but keeping the height from getting too big would then stop all the other 'tabs' from getting pushed down and out of site as well. This is how it worked in V7, so editors may well be used to the RTE scroll bar.

Regards
Iain Martin

**Comment by nul800sebastiaan** [comment]
Thanks @IainMartin-Ayr - we must have overlooked something in v8!

To confirm:
- When you set a width and a height for the RTE datatype, it should reflect on the content item, currently in v8 it does not
- When a height is set, a scrollbar should appear inside the RTE when you add content that is higher, currently the RTE just grows downwards.

**Comment by IainMartin-Ayr** [comment]
Thanks Seb, your confirmation is spot on. It wasn't until I had put in a large amount of text that I realised that the RTE was growing and pushing everything down with it. And as I'm prone to break things with my own CSS at times reckoned I should check that possibility first.

I am also using the U8Collapse plugin by Simon Campbell to show/hide the panes and I also managed to eliminate that option as well just in case that was interfering with things.

And now that I've got over most of the 'how do I do that in V8' questions, really starting to like it a lot 👍
